test.js

```diff
@@ -58,3 +70,15 @@ test('Get ID by German name', testIDs, 'de', [
 	'Bidifas',
 	'Volcanion'
 ]);
+
+test('Get ID by Chinese name', testIDs, 'zh-CN', [
+	'妙蛙种子',
+	'大尾狸',
+	'ボルケニオン'
+]);
+
+test('Get ID by Japanese name', testIDs, 'jp', [
+	'フシギダネ',
+	'ビーダル',
+	'ボルケニオン'
+]);
```

Comment:
We don't need the name test for all languages. We're already testing that code with English and German. We just need the other ID test to ensure the language works, so these two tests can be dropped.